docs/docs/api/LayoutAnimations/CustomAnimations.md

```diff
@@ -0,0 +1,196 @@
+    * `targetValues.originX` - X coordinate of top left corder in parent's coordinate system
+    * `targetValues.originY` - Y coordinate of top left corder in parent's coordinate system
+    * `targetValues.width` - view's width
+    * `targetValues.height` -view's height
+    * `targetValues.globalOriginX` - X coordinate of top left corder in global coordinate system
+    * `targetValues.globalOriginY` - Y coordinate of top left corder in global coordinate system
+
+### Example
+```js
+
+function CardView() {
+  const entering = (targetValues) => {
+    'worklet';
+    const animations = {
+      originX: withTiming(targetValues.originX, { duration: 3000 }),
+      opacity: withTiming(1, { duration: 2000 }),
+      borderRadius: withDelay(4000, withTiming(30, { duration: 3000 })),
+      transform: [
+        { rotate: withTiming('0deg', { duration: 4000 }) },
+        { scale: withTiming(1, { duration: 3500 }) },
+      ],
+    };
+    const initialValues = {
+      originX: -width,
+      opacity: 0,
+      borderRadius: 10,
+      transform: [{ rotate: '90deg' }, { scale: 0.5 }],
+    };
+    return {
+      initialValues,
+      animations,
+    };
+  };
+
+
+  return (
+    <Animated.View
+      style={[styles.animatedView]} entering={entering} >
+      <Text> Card Example </Text>
+    </Animated.View>
+  );
+}
+
+```
+
+## Custom Layout Transtion
+
+What our layout transition builders do under the hood is generating a worklet function that returns essential data for starting particular transition. 
+The high level template looks like this:
+
+```js
+function CustomLayoutTransition(values) {
+    'worklet'
+    const animations = {
+        // your animations
+    };
+    const initialValues = {
+        // initial values for animations
+    };
+    return {
+      initialValues,
+      animations,
+    }
+}
+```
+
+* `values` - contains before and after information about the view's origin and dimations
+    * `values.originX` - X coordinate of top left corder in parent's coordinate system (after)
+    * `values.originY` - Y coordinate of top left corder in parent's coordinate system (after)
+    * `values.width` - view's width (after)
+    * `values.height` -view's height (after)
+    * `values.globalOriginX` - X coordinate of top left corder in global coordinate system (after)
+    * `values.globalOriginY` - Y coordinate of top left corder in global coordinate system (after)
+    * `values.boriginX` - X coordinate of top left corder in parent's coordinate system (before)
+    * `values.boriginY` - Y coordinate of top left corder in parent's coordinate system (before)
+    * `values.bwidth` - view's width (before)
+    * `values.bheight` -view's height (before)
+    * `values.bglobalOriginX` - X coordinate of top left corder in global coordinate system (before)
+    * `values.bglobalOriginY` - Y coordinate of top left corder in global coordinate system (before)
+
+### Example
+```js
+
+function CardView() {
```

Comment:
It would be good to show the style of the element here. From the example, I cannot infer what is the initial state and the end state, maybe we should make it explicit in a comment.